static/style.css

```
@@ -0,0 +1,39 @@
+* {
+    margin: 0; padding: 0; box-sizing: border-box;
+    font-family: 'Segoe UI', Tahoma, italic serif;
```

Stylelint (17.2.0)

[error] Expected quotes around "Tahoma serif" (font-family-name-quotes)

(font-family-name-quotes)